cmd/deploy/deploy.go

```diff
@@ -206,6 +211,12 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		}
 	}
 
+	if os.Getenv(model.OktetoWithinDeployCommandContextEnvVar) != "" {
```

Comment:
should this be `== ""`? we show endpoints if it's not within a deploy command

Reply:
I was considering only if it was not set as if it was set to false, but yes, this should be ==